src/api/routes/status.ts

```diff
@@ -0,0 +1,23 @@
+import { TypeBoxTypeProvider } from "@fastify/type-provider-typebox";
+import { Type } from "@sinclair/typebox";
+import { FastifyPluginCallback } from "fastify";
+import { Server } from "http";
+
+export const StatusRoutes: FastifyPluginCallback<
+  Record<never, never>,
+  Server,
+  TypeBoxTypeProvider
+> = (fastify, options, done) => {
+  fastify.get('/', {
```

Comment:
I think it makes sense for some route (either `/` or `/status` to provide a simple response like this just to indicate to docker or whatever tooling that the service is online, but should there also be an endpoint that gives the status of what is currently going on in the service (like related to jobs in process, progress percent, etc)?